### ISSUE TITLE
Setup fail during integration loading

### ISSUE THREAD

**Issue by tgerardy** [p0l0/hapetwalk#23]
## System Information

version | core-2024.1.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.11.6
os_name | Linux
os_version | 6.1.71-haos
arch | x86_64
timezone | Europe/Paris
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.33.0
Stage | running
Available Repositories | 1386
Downloaded Repositories | 40

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 21 novembre 2024 à 01:00
relayer_connected | true
relayer_region | eu-central-1
remote_enabled | true
remote_connected | true
alexa_enabled | true
google_enabled | true
remote_server | eu-central-1-4.ui.nabu.casa
certificate_status | ready
instance_id | 2eb688bb73a743f0b79ae403172c96d4
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 11.4
-- | --
update_channel | stable
supervisor_version | supervisor-2023.12.0
agent_version | 1.6.0
docker_version | 24.0.7
disk_total | 30.8 GB
disk_used | 25.0 GB
healthy | true
supported | true
board | ova
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.7.0), Samba share (12.2.0), Studio Code Server (5.15.0), Mosquitto broker (6.4.0), Advanced SSH & Web Terminal (17.0.3), Home Assistant Google Drive Backup (0.112.1), Z-Wave JS (0.4.3), Ring-MQTT with Video Streaming (5.6.3), RTSPtoWeb - WebRTC (1.4.0), Zigbee2MQTT (1.35.1-1), Scrypted (18-jammy-full.s6-v0.84.0), Node-RED (17.0.2)

</details>

<details><summary>Dashboards</summary>

dashboards | 4
-- | --
resources | 23
views | 39
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 5 janvier 2024 à 19:03
-- | --
current_recorder_run | 15 janvier 2024 à 23:27
estimated_db_size | 1068.34 MiB
database_engine | sqlite
database_version | 3.41.2

</details>

<details><summary>Spotify</summary>

api_endpoint_reachable | ok
-- | --

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of any [previous issues](https://github.com/p0l0/hapetwalk/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

Hi, thank you for developing this integration. I didn't expect anyone to develop such an integration since Petwalk doors are not a very widespread product. Great to see you made it. Unfortunately, the integration fails for me at loading with the message "setup fails: unknown property value". 

### Reproduction steps

1. Integration fails to load with message "setup fails: unknown property value"
2. I am using the credentials which I use for the control app or the online api
3. The IP address is fixed
...

### Debug logs

```text
2024-01-16 00:25:37.609 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration govee_lan which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.610 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration tahoma which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.610 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration nodered which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.612 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration wiser which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.613 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration alexa_media which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.614 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration ui_lovelace_minimalist which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.614 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration keyatome which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.615 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration roborock which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.616 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.616 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration climate_group which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.617 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration zha_toolkit which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.617 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration eero which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.618 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration petwalk which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.618 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration webrtc which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.619 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration nuki_ng which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.620 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration dwains_dashboard which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.620 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration average which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.621 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration scheduler which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:37.621 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration xiaomi_cloud_map_extractor which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-01-16 00:25:47.546 WARNING (MainThread) [homeassistant.components.climate.const] HVAC_MODE_OFF was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACMode.OFF instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.550 WARNING (MainThread) [homeassistant.components.climate.const] HVAC_MODE_HEAT was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACMode.HEAT instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.555 WARNING (MainThread) [homeassistant.components.climate.const] HVAC_MODE_COOL was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACMode.COOL instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.560 WARNING (MainThread) [homeassistant.components.climate.const] HVAC_MODE_HEAT_COOL was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACMode.HEAT_COOL instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.564 WARNING (MainThread) [homeassistant.components.climate.const] HVAC_MODE_AUTO was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACMode.AUTO instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.569 WARNING (MainThread) [homeassistant.components.climate.const] HVAC_MODE_DRY was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACMode.DRY instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.573 WARNING (MainThread) [homeassistant.components.climate.const] HVAC_MODE_FAN_ONLY was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACMode.FAN_ONLY instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.578 WARNING (MainThread) [homeassistant.components.climate.const] CURRENT_HVAC_OFF was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACAction.OFF instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.583 WARNING (MainThread) [homeassistant.components.climate.const] CURRENT_HVAC_HEAT was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACAction.HEATING instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.588 WARNING (MainThread) [homeassistant.components.climate.const] CURRENT_HVAC_COOL was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACAction.COOLING instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.592 WARNING (MainThread) [homeassistant.components.climate.const] CURRENT_HVAC_DRY was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACAction.DRYING instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.598 WARNING (MainThread) [homeassistant.components.climate.const] CURRENT_HVAC_IDLE was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACAction.IDLE instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.603 WARNING (MainThread) [homeassistant.components.climate.const] CURRENT_HVAC_FAN was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use HVACAction.FAN instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.608 WARNING (MainThread) [homeassistant.components.climate.const] SUPPORT_TARGET_TEMPERATURE was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.TARGET_TEMPERATURE instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.613 WARNING (MainThread) [homeassistant.components.climate.const] SUPPORT_TARGET_TEMPERATURE_RANGE was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.TARGET_TEMPERATURE_RANGE instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.620 WARNING (MainThread) [homeassistant.components.climate.const] SUPPORT_TARGET_HUMIDITY was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.TARGET_HUMIDITY instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.625 WARNING (MainThread) [homeassistant.components.climate.const] SUPPORT_FAN_MODE was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.FAN_MODE instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.630 WARNING (MainThread) [homeassistant.components.climate.const] SUPPORT_PRESET_MODE was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.PRESET_MODE instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.636 WARNING (MainThread) [homeassistant.components.climate.const] SUPPORT_SWING_MODE was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.SWING_MODE instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.642 WARNING (MainThread) [homeassistant.components.climate.const] SUPPORT_AUX_HEAT was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.AUX_HEAT instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.647 WARNING (MainThread) [homeassistant.const] TEMP_CELSIUS was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.652 WARNING (MainThread) [homeassistant.const] TEMP_FAHRENHEIT was used from climate_group, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.FAHRENHEIT instead, please report it to the author of the 'climate_group' custom integration
2024-01-16 00:25:47.840 ERROR (MainThread) [homeassistant.setup] Setup failed for 'powercalc': Integration not found.
2024-01-16 00:25:52.466 WARNING (MainThread) [homeassistant.components.lock] SUPPORT_OPEN was used from nuki_ng, this is a deprecated constant which will be removed in HA Core 2025.1. Use LockEntityFeature.OPEN instead, please create a bug report at https://github.com/kvj/hass_nuki_ng/issues
2024-01-16 00:25:52.473 WARNING (MainThread) [homeassistant.components.lock] SUPPORT_OPEN was used from nuki_ng, this is a deprecated constant which will be removed in HA Core 2025.1. Use LockEntityFeature.OPEN instead, please create a bug report at https://github.com/kvj/hass_nuki_ng/issues
2024-01-16 00:25:57.318 WARNING (SyncWorker_5) [pynuki] Call failed: <Response [200]>
2024-01-16 00:25:59.984 WARNING (MainThread) [homeassistant.helpers.entity] Entity climate.radiateur_tous (<class 'custom_components.climate_group.climate.ClimateGroup'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <ClimateEntityFeature: 0>, please report it to the author of the 'climate_group' custom integration and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation
2024-01-16 00:26:00.704 WARNING (MainThread) [homeassistant.components.camera] SUPPORT_ON_OFF was used from xiaomi_cloud_map_extractor, this is a deprecated constant which will be removed in HA Core 2025.1. Use CameraEntityFeature.ON_OFF instead, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues
2024-01-16 00:26:00.713 WARNING (MainThread) [homeassistant.components.camera] SUPPORT_ON_OFF was used from xiaomi_cloud_map_extractor, this is a deprecated constant which will be removed in HA Core 2025.1. Use CameraEntityFeature.ON_OFF instead, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues
2024-01-16 00:26:04.932 WARNING (MainThread) [homeassistant.helpers.entity] Entity camera.xiaomi_cloud_map_extractor (<class 'custom_components.xiaomi_cloud_map_extractor.camera.VacuumCamera'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <CameraEntityFeature.ON_OFF: 1>, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation
2024-01-16 00:26:06.751 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x0004740000c582e1'], 'manufacturer': 'Legrand', 'model': 'Wireless shutter switch (067646)', 'name': '0x0004740000c582e1', 'sw_version': '002a', 'connections': []}, 'device_class': <BinarySensorDeviceClass.UPDATE: 'update'>, 'enabled_by_default': False, 'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>, 'name': '0x0004740000c582e1 update available', 'payload_off': 'False', 'payload_on': 'True', 'state_topic': 'zigbee2mqtt/0x0004740000c582e1', 'unique_id': '0x0004740000c582e1_update_available_zigbee2mqtt', 'value_template': Template<template=({{ value_json['update']['state'] == "available" }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'update available'
2024-01-16 00:26:06.752 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x00047400007a991b'], 'manufacturer': 'Legrand', 'model': 'Wireless remote switch (067773)', 'name': '0x00047400007a991b', 'sw_version': '003c', 'connections': []}, 'device_class': <SensorDeviceClass.BATTERY: 'battery'>, 'enabled_by_default': True, 'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>, 'name': '0x00047400007a991b battery', 'state_class': <SensorStateClass.MEASUREMENT: 'measurement'>, 'state_topic': 'zigbee2mqtt/0x00047400007a991b', 'unique_id': '0x00047400007a991b_battery_zigbee2mqtt', 'unit_of_measurement': '%', 'value_template': Template<template=({{ value_json.battery }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'battery'
2024-01-16 00:26:06.753 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x00047400007a991b'], 'manufacturer': 'Legrand', 'model': 'Wireless remote switch (067773)', 'name': '0x00047400007a991b', 'sw_version': '003c', 'connections': []}, 'enabled_by_default': True, 'icon': 'mdi:gesture-double-tap', 'name': '0x00047400007a991b action', 'state_topic': 'zigbee2mqtt/0x00047400007a991b', 'unique_id': '0x00047400007a991b_action_zigbee2mqtt', 'value_template': Template<template=({{ value_json.action }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'action'
2024-01-16 00:26:06.754 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x00047400007a991b'], 'manufacturer': 'Legrand', 'model': 'Wireless remote switch (067773)', 'name': '0x00047400007a991b', 'sw_version': '003c', 'connections': []}, 'enabled_by_default': False, 'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>, 'icon': 'mdi:signal', 'name': '0x00047400007a991b linkquality', 'state_class': <SensorStateClass.MEASUREMENT: 'measurement'>, 'state_topic': 'zigbee2mqtt/0x00047400007a991b', 'unique_id': '0x00047400007a991b_linkquality_zigbee2mqtt', 'unit_of_measurement': 'lqi', 'value_template': Template<template=({{ value_json.linkquality }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'linkquality'
2024-01-16 00:26:06.755 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x0004740000c582e1'], 'manufacturer': 'Legrand', 'model': 'Wireless shutter switch (067646)', 'name': '0x0004740000c582e1', 'sw_version': '002a', 'connections': []}, 'device_class': <SensorDeviceClass.BATTERY: 'battery'>, 'enabled_by_default': True, 'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>, 'name': '0x0004740000c582e1 battery', 'state_class': <SensorStateClass.MEASUREMENT: 'measurement'>, 'state_topic': 'zigbee2mqtt/0x0004740000c582e1', 'unique_id': '0x0004740000c582e1_battery_zigbee2mqtt', 'unit_of_measurement': '%', 'value_template': Template<template=({{ value_json.battery }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'battery'
2024-01-16 00:26:06.755 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x0004740000c582e1'], 'manufacturer': 'Legrand', 'model': 'Wireless shutter switch (067646)', 'name': '0x0004740000c582e1', 'sw_version': '002a', 'connections': []}, 'enabled_by_default': True, 'icon': 'mdi:gesture-double-tap', 'name': '0x0004740000c582e1 action', 'state_topic': 'zigbee2mqtt/0x0004740000c582e1', 'unique_id': '0x0004740000c582e1_action_zigbee2mqtt', 'value_template': Template<template=({{ value_json.action }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'action'
2024-01-16 00:26:06.756 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x0004740000c582e1'], 'manufacturer': 'Legrand', 'model': 'Wireless shutter switch (067646)', 'name': '0x0004740000c582e1', 'sw_version': '002a', 'connections': []}, 'enabled_by_default': False, 'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>, 'icon': 'mdi:signal', 'name': '0x0004740000c582e1 linkquality', 'state_class': <SensorStateClass.MEASUREMENT: 'measurement'>, 'state_topic': 'zigbee2mqtt/0x0004740000c582e1', 'unique_id': '0x0004740000c582e1_linkquality_zigbee2mqtt', 'unit_of_measurement': 'lqi', 'value_template': Template<template=({{ value_json.linkquality }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'linkquality'
2024-01-16 00:26:06.758 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT entity name starts with the device name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'device': {'identifiers': ['zigbee2mqtt_0x0004740000c582e1'], 'manufacturer': 'Legrand', 'model': 'Wireless shutter switch (067646)', 'name': '0x0004740000c582e1', 'sw_version': '002a', 'connections': []}, 'enabled_by_default': False, 'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>, 'icon': 'mdi:update', 'name': '0x0004740000c582e1 update state', 'state_topic': 'zigbee2mqtt/0x0004740000c582e1', 'unique_id': '0x0004740000c582e1_update_state_zigbee2mqtt', 'value_template': Template<template=({{ value_json['update']['state'] }}) renders=0>, 'force_update': False, 'encoding': 'utf-8', 'payload_available': 'online', 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The device name prefix will be stripped off the entity name and becomes 'update state'
2024-01-16 00:26:06.764 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT device name is equal to entity name in your config {'availability': [{'topic': 'zigbee2mqtt/bridge/state', 'value_template': Template<template=({{ value_json.state }}) renders=0>, 'payload_not_available': 'offline', 'payload_available': 'online'}], 'command_topic': 'zigbee2mqtt/bridge/request/device/ota_update/update', 'device': {'identifiers': ['zigbee2mqtt_0x0004740000c582e1'], 'manufacturer': 'Legrand', 'model': 'Wireless shutter switch (067646)', 'name': '0x0004740000c582e1', 'sw_version': '002a', 'connections': []}, 'device_class': <UpdateDeviceClass.FIRMWARE: 'firmware'>, 'entity_picture': 'https://github.com/Koenkk/zigbee2mqtt/raw/master/images/logo.png', 'latest_version_template': Template<template=({{ value_json['update']['latest_version'] }}) renders=0>, 'latest_version_topic': 'zigbee2mqtt/0x0004740000c582e1', 'name': '0x0004740000c582e1', 'payload_install': '{"id": "0x0004740000c582e1"}', 'state_topic': 'zigbee2mqtt/0x0004740000c582e1', 'unique_id': '0x0004740000c582e1_update_zigbee2mqtt', 'value_template': Template<template=({{ value_json['update']['installed_version'] }}) renders=0>, 'encoding': 'utf-8', 'payload_available': 'online', 'enabled_by_default': True, 'retain': False, 'payload_not_available': 'offline', 'qos': 0, 'availability_mode': 'latest'}, this is not expected. Please correct your configuration. The entity name will be set to `null`
2024-01-16 00:26:07.438 ERROR (MainThread) [custom_components.petwalk.coordinator] Unexpected error fetching petwalk data: Unknown property: value
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 300, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/petwalk/coordinator.py", line 158, in _async_update_data
    data[COORDINATOR_KEY_PET_STATUS] = await self._api.get_pet_status(
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pypetwalk/pypetwalk.py", line 175, in get_pet_status
    event = Event(entry)
            ^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pypetwalk/aws/event.py", line 39, in __init__
    raise ValueError(f"Unknown property: {key}")
ValueError: Unknown property: value
2024-01-16 00:26:10.096 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/withings/__init__.py", line 245, in register_webhook
    await async_subscribe_webhooks(client, webhook_url)
  File "/usr/src/homeassistant/homeassistant/components/withings/__init__.py", line 309, in async_subscribe_webhooks
    await client.subscribe_notification(webhook_url, notification)
  File "/usr/local/lib/python3.11/site-packages/aiowithings/withings.py", line 349, in subscribe_notification
    await self._request(
  File "/usr/local/lib/python3.11/site-packages/aiowithings/withings.py", line 133, in _request
    raise WithingsInvalidParamsError(error)
aiowithings.exceptions.WithingsInvalidParamsError: The callback URL is either absent or incorrect
2024-01-16 00:26:10.156 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/withings/__init__.py", line 245, in register_webhook
    await async_subscribe_webhooks(client, webhook_url)
  File "/usr/src/homeassistant/homeassistant/components/withings/__init__.py", line 309, in async_subscribe_webhooks
    await client.subscribe_notification(webhook_url, notification)
  File "/usr/local/lib/python3.11/site-packages/aiowithings/withings.py", line 349, in subscribe_notification
    await self._request(
  File "/usr/local/lib/python3.11/site-packages/aiowithings/withings.py", line 133, in _request
    raise WithingsInvalidParamsError(error)
aiowithings.exceptions.WithingsInvalidParamsError: The callback URL is either absent or incorrect
2024-01-16 00:26:12.577 WARNING (SyncWorker_5) [custom_components.xiaomi_cloud_map_extractor.camera] Got exception while fetching the state: Unable to discover the device 192.168.4.150
2024-01-16 00:26:13.671 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.lixee_zipulses_eau_summation_delivered (<class 'homeassistant.components.zha.sensor.SmartEnergySummation'>) is using native unit of measurement 'L' which is not a valid unit for the device class ('energy') it is using; expected one of ['MWh', 'Wh', 'kWh', 'MJ', 'GJ']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+zha%22
2024-01-16 00:26:14.937 WARNING (MainThread) [homeassistant.helpers.entity] Update of camera.xiaomi_cloud_map_extractor is taking over 10 seconds
2024-01-16 00:27:04.965 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:09.965 WARNING (MainThread) [homeassistant.helpers.entity] Update of camera.xiaomi_cloud_map_extractor is taking over 10 seconds
2024-01-16 00:27:09.966 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:14.967 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:19.969 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:24.970 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:29.971 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:34.972 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:39.974 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:44.976 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:49.976 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-01-16 00:27:51.076 WARNING (MainThread) [homeassistant.components.group.sensor] Unable to use state. Only numerical states are supported, entity sensor.total_chauffage_instant_power with value unavailable excluded from calculation
```

**Comment by tgerardy** [comment]
Thank you for the quick feedback.

**Comment by p0l0** [comment]
Issue should now be fixed in version 2.0.2

**Comment by tgerardy** [comment]
Well done! That works now. Thanks a lot.
